crates/rpc/src/v08.rs

```diff
@@ -5,6 +5,9 @@ use crate::method::subscribe_pending_transactions::SubscribePendingTransactions;
 #[rustfmt::skip]
 pub fn register_routes() -> RpcRouterBuilder {
     RpcRouter::builder(crate::RpcVersion::V08)
+        // TODO This is for testing - figure out if this should be removed or included when
+        // implementing RPC spec 0.8.
```

Comment:
It should _definitely_ be included and there no changes between 0.7.1 and 0.8.0-rc0 in `starknet_syncing` so it should be just fine to add this method.